src/app/pages/MediaAssetPage/index.stories.jsx

```diff
@@ -24,6 +24,11 @@ export default {
   Component,
   title: 'Pages/Media Asset Page',
   decorators: [story => <WithTimeMachine>{story()}</WithTimeMachine>],
+  parameters: {
+    chromatic: {
+      delay: 15000,
```

Comment:
Looks like this is still flaky, but I believe this is the longest delay you can set...? Should we consider trying to ignore the related content images on the media asset page?

Reply:
We have another ticket to ignore images entirely, I guess we could just ignore images on selected stories couldn't we?

Reply:
Let's leave the image issue for the other ticket - this is already a vast improvement, and if we only have to approve 1 flaky UI test, that's already better 😸

Reply:
I'll drop a note on the ticket to suggest we try only flakey stories initially